packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/stack.md

```diff
@@ -2,4 +2,31 @@
 id: Stack
 section: layouts
 ---
-Use a **Stack** layout to position items vertically in a container, with one item filling the available vertical space.
+The **stack** layout positions items vertically, with one or more items filling the available vertical space.
+
+## Elements
+
+<img src="./img/stack-example.png"  alt="PatternFly website screenshot showing modal created with bullseye layout."  width="3330px"/>
+
+1. **Stack**: The stack container, which holds all stack items.
+1. **Stack item**: An individual section of content to be placed in the stack. 
+
+## Usage
+
+### When to use stack vs other layouts 
+
+The stack layout does not wrap. If you need a layout that wraps, instead choose the flex, gallery, level, or split layout instead.
+
+## Variations
+
+### Multiple items that fill space 
+
+You can specify which stack item fill the remaining vertical space in the layout. More than one item can be set to fill and they will share the remaining space equally. 
```

Comment:
Should this say which stack item fill**S**, not item fill? or am I reading it wrong